apps/traefik-certs-dumper/docker-compose.yml

```
@@ -0,0 +1,23 @@
+version: "3.8"
+
+services:
+  traefik-certs-dumper:
+    container_name: traefik-certs-dumper
+    # user: root
+    image: humenius/traefik-certs-dumper:1.6.1-alpine
+    restart: unless-stopped
+    healthcheck:
+      test: ["CMD", "/usr/bin/healthcheck"]
+      interval: 30s
+      timeout: 10s
+      retries: 5
+    volumes:
+      - /root/runtipi/traefik/shared/acme.json:/traefik/acme.json:ro
```

This will not work with everyone as runtipi could be installed in any folder. You can replace by

```suggestion
      - ${ROOT_FOLDER_HOST}/traefik/shared/acme.json:/traefik/acme.json:ro
```

---

Thanks forgotten about it in this compose !